fips/usr/lib/systemd/system/snap-initramfs-mounts.service.d/fips.conf

```diff
@@ -0,0 +1,10 @@
+[Unit]
+After=systemd-modules-load.service
+
+[Service]
+# Ensure tcrypt was loaded
+ExecStartPre=modprobe tcrypt
+# Then allow snap-bootstrap to mount things
+# Self-test verify FIPS computed hmac
+ExecStartPost=sha512hmac --check /run/mnt/kernel/.kernel.efi.hmac
```

Comment:
Where does sha512hmac come from? I installed `kcapi-tools` but it does not look to be in it.

Reply:
Correct, only fips compliant version of kcapi-tools ships that binary in Focal & Jammy today. And this initrd must only be build against fips compliant versions of kernels & kcapi-tools. (I.e. enabling this feature alone does not make ones' kerne.efi fips compliant, one actually needs linux-fips kernel).

For mantic+ releases kcapi-hasher is shipped and will be used instead.